lib/utils/report-deprecated-properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const _ = require('lodash');
+const logDeprecation = require('./logDeprecation');
+
+module.exports = (code, props, { serviceConfig } = {}) => {
+  const warnings = [];
+
+  for (const [oldProp, newProp] of Object.entries(props)) {
+    if (_.has(serviceConfig, oldProp)) {
+      warnings.push(
+        `Starting with version 3.0.0, "${oldProp}" property will be replaced with "${newProp}"`
```

Comment:
Thanks @runk for update. In general it looks good, I will just improve messaging, as when constructed as proposed itwill create a lot of repetitive noise (_"Starting with... Starting with... Starting with..."_)

Ideally if we simply group names of deprecated properties, and then construct one concise message as:

```
Starting with v3.0.0, support for "a", "b" and "c" will be removed. Please use "d", "e" and "f" instead
```

Reply:
Okay, I used slightly different formatting, please check (as well as tests)

cc @medikoo